omx/gstomxaacdec.c

```diff
@@ -79,87 +78,125 @@ gst_omx_aac_dec_class_init (GstOMXAACDecClass * klass)
 static void
 gst_omx_aac_dec_init (GstOMXAACDec * self)
 {
-  /* FIXME: Other values exist too! */
-  self->spf = 1024;
+  g_return_val_if_fail (self, NULL);
```

Comment:
This should give you a warning? Does this compile?

Reply:
Yes it compiled that way, didn't catch the warnings at build time. Here 0947ed4 the g_return_val_if_fail function incorrect usage is addressed. Please review